make_queue/templates/make_queue/course/registration_edit.html

```diff
@@ -32,6 +33,16 @@ <h1>
                 </div>
                 <p> {% trans "Are you sure you entered the correct username?" %} </p>
             </div>
+            {% form_has_error form "username" "unique" as unique_error %}
+            {% if unique_error %}
+                <div class="ui error message">
+                    <i class="close icon"></i>
+                    <div class="header">
+                        {% trans "Username already exists" %}
+                    </div>
+                    <p>{% trans "Are you sure you entered the correct username?" %}</p>
```

Comment:
Should we not have some other message than this in the description part of the unique constraint error. Something along the lines of "A course registration for the given username already exists. Either the user has already taken the course, or the username of another user has been entered."

Reply:
Yes, that gives the user more information.